core/sdk-core/src/main/java/software/amazon/awssdk/core/internal/http/RequestExecutionContext.java

```diff
@@ -124,6 +127,10 @@ public void requestProvider(AsyncRequestBody publisher) {
         requestProvider = publisher;
     }
 
+    public CompletableFuture streamingAsyncResponseTransformerFuture() {
```

Comment:
Let's use wild card `CompletableFuture<?>` instead of raw type.
`asyncResponseTransformerFuture()` if we decide to drop the prefix

Reply:
Done

Reply:
Compiler is not happy with wild card when assigning it to generic field. Reverted back to raw type

Reply:
Hmm, are we using raw type CompletableFuture any where else?

Reply:
No, this is the only place afaik. Compiler error [here](https://github.com/aws/aws-sdk-java-v2/blob/565175424144cf063bcd61df484a55125ccb5fca/core/sdk-core/src/main/java/software/amazon/awssdk/core/internal/http/pipeline/stages/MakeAsyncHttpRequestStage.java#L95) if wild card is used.